tests/Console/ModelsCommand/GenerateBasicPhpdocLower/Test.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\GenerateBasicPhpdocLower;
+
+use Barryvdh\LaravelIdeHelper\Console\ModelsCommand;
+use Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\AbstractModelsCommand;
+
+class Test extends AbstractModelsCommand
+{
+    protected function getEnvironmentSetUp($app)
+    {
+        parent::getEnvironmentSetUp($app);
+
+        $app['config']->set('ide-helper.model_lower_case_properties', true);
+    }
+
+    public function test(): void
```

Comment:
This test doesn't really perform the test you intended it to do, because none of the migrations provide upper case column definitions.

Reply:
Added proper tests by creating column names that are uppercase